R4ML/R/ml.preprocess.R

```diff
@@ -596,3 +596,32 @@ isAnyAttrOfType <- function(hm, attrs, type) {
   return(any(allColumnTypes[allColumnNames %in% attrs] == type))
 }
 
+# Method to transform a r4ml.frame to a r4ml.matrix that can be read by DML 
+#'
+#' @name r4ml.systemml.transform
+#' @title Wrapper method for \code{\link{r4ml.ml.preprocess}}
+#' @description Wrapper method for \code{\link{r4ml.ml.preprocess}} and produces a \code{r4ml.matrix}.
```

Comment:
I think somehow in the documentation we should tell the user that the input dataset has to be numeric